### PR TITLE
Rename rigid body type Dynamic to Simulated.

### PR DESCRIPTION
**Pull request by moraaar**
Signed-off-by: moraaar <moraaar@amazon.com>

## What does this PR do?

This is to avoid collision with the renaming of PhysX Rigid Body to PhysX Dynamic Rigid Body.

## How was this PR tested?

Run PhysX unit tests.
Tested having a previous dynamic type in a level will correctly stay as simulated after the code change.

image

image

### CODE REVIEW

**Review by greerdv — APPROVED** [review]

File: Gems/PhysX/Code/Editor/KinematicDescriptionDialog.ui [hunk 1]
```diff
@@ -115,7 +115,7 @@
             <string notr="true">border: none;</string>
            </property>
            <property name="text">
-            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Dynamic&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Simulated&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```
Comment (moraaar) [inline Gems/PhysX/Code/Editor/KinematicDescriptionDialog.ui:118]: This label is appearing as "Simulate" in the UI, I'll increment the width so the whole word fits.

**Comment by greerdv** [comment]
In the screenshot it seems like it just says "Simulate" rather than "Simulated".

Can we change "physics are applied" to "physics is applied"? Physics is usually treated as a singular.

**Comment by moraaar** [comment]
> Can we change "physics are applied" to "physics is applied"? Physics is usually treated as a singular.

Will change. Thanks.

**Comment by moraaar** [comment]
> In the screenshot it seems like it just says "Simulate" rather than "Simulated".
> 
> Can we change "physics are applied" to "physics is applied"? Physics is usually treated as a singular.

Done and done!

File: Gems/PhysX/Code/Editor/KinematicDescriptionDialog.ui (hunk 1 above)
Comment (moraaar) [reply Gems/PhysX/Code/Editor/KinematicDescriptionDialog.ui:118]: Fixed

**Review by hultonha — APPROVED** [review]
Great change, thanks for being so thorough and fixing all comments and variables too 👍

**Pull request merged**